devel/ld64/Portfile

```diff
@@ -162,13 +170,26 @@ subport ld64-latest {
         ld64-134-missing-include.patch \
         ld64-136-i386-badAddress.patch \
         ld64-ppc-9610466.patch \
+        ld64-274-use-after-free.patch \
         PR-49393.patch \
         PR-28988396.patch \
         PR-29117886.patch \
-        PR-29679726.patch \
         PR-29723276.patch \
+        PR-33746767.patch \
         PR-29723629.patch \
-        PR-33746767.patch
+        PR-30596832.patch
+        
+
+    # TAPI fixes:
+    
+    # only +llvm37 is presently known to work with tapi-1.30 code base 
+    # llvm34 fails, llvm39 fails, llvm40 fails, llvm50 fails
+    default_variants +llvm37
+
+    # patch out missing definitions for TVOS and WATCHOS not in llvm-3.7 include files
+    post-extract {
+        system "cd ${workpath}/libtapi-${tapi_version} && patch -p1 < ${filespath}/patch-libtapi-lib-Core-MachODylibReader-unknown-LC-options.diff"
```

Comment:
Use `system -W ${workpath}/libtapi-${tapi_version} "patch ..."`, not `system "cd ${workpath}/libtapi-${tapi_version} && patch ...`.